tests/console/java.pm

```diff
@@ -20,7 +20,42 @@ use utils;
 use base "consoletest";
 sub run() {
     select_console 'root-console';
-    zypper_call "in java-*";
+
+    # Make sure that PackageKit is not running
+    pkcon_quit;
+
+    # Capture the return code value of the following scenarios
+    my $bootstrap_pkg_rt       = zypper_call("se java-*bootstrap",  exitcode => [0, 104]);
+    my $bootstrap_conflicts_rt = zypper_call("in --dry-run java-*", exitcode => [0, 4]);
+
+    # logs / debugging purposes
+    diag "checking variable: bootstrap_pkg_rt = $bootstrap_pkg_rt";
+    diag "checking variable: bootstrap_conflicts_rt = $bootstrap_conflicts_rt";
+
+    if (check_var("DISTRI", "sle")) {
+        zypper_call "in java-*";
+    }
+
+    if (check_var("DISTRI", "opensuse")) {
+        if ($bootstrap_pkg_rt == 0) {
+            print "There are java bootstrap packages available to be installed\n";
```

Comment:
no, I don't want to replace it, I need this to be visible in the screenshots. However, I can 'add' a diag, so this will be also visible in the logs.